root/programs/MultiPurposeAuthSite/MultiPurposeAuthSite/Models/ASPNETIdentity/TokenProviders/AccessTokenFormatJwt.cs

```diff
@@ -0,0 +1,297 @@
+﻿//**********************************************************************************
+//* Copyright (C) 2007,2016 Hitachi Solutions,Ltd.
+//**********************************************************************************
+
+#region Apache License
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License. 
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+#endregion
+
+//**********************************************************************************
+//* クラス名        ：AccessTokenFormatJwt
+//* クラス日本語名  ：Access TokenのJWTカスタマイズクラス
+//*
+//* 作成日時        ：－
+//* 作成者          ：－
+//* 更新履歴        ：－
+//*
+//*  日時        更新者            内容
+//*  ----------  ----------------  -------------------------------------------------
+//*  2017/04/24  西野 大介         新規
+//**********************************************************************************
+
+using System;
+using System.Web;
+using System.Security.Claims;
+using System.Collections.Generic;
+using System.Security.Cryptography.X509Certificates;
+
+using Microsoft.Owin.Security;
+using Microsoft.Owin.Security.OAuth;
+using Microsoft.AspNet.Identity;
+using Microsoft.AspNet.Identity.Owin;
+
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+
+using MultiPurposeAuthSite.Models.ASPNETIdentity.Manager;
+using MultiPurposeAuthSite.Models.ASPNETIdentity.Entity;
+
+using Touryo.Infrastructure.Public.Str;
+using Touryo.Infrastructure.Public.Util.JWT;
+
+namespace MultiPurposeAuthSite.Models.ASPNETIdentity.TokenProviders
+{
+    /// <summary>AccessTokenFormatJwt</summary>
+    public class AccessTokenFormatJwt: ISecureDataFormat<AuthenticationTicket>
+    {
+        ///// <summary>IssuerId</summary>
+        //private readonly string _oAuthIssuerId = string.Empty;
+
+        /// <summary>constructor</summary>
+        /// <param name="oAuthIssuerId">oAuthIssuerId</param>
+        public AccessTokenFormatJwt()
+        {
+        }
+
+        /// <summary>Protect</summary>
+        /// <param name="data">AuthenticationTicket</param>
```

Comment:
@daisukenishino 

Since the type of this variable is `AuthenticationTicket`, it might be a good idea to name this variable *ticket*.